hibernate-core/src/main/java/org/hibernate/dialect/Dialect.java

```diff
@@ -2864,6 +2864,24 @@ public boolean supportsNationalizedTypes() {
 		return true;
 	}
 
+	/**
+	 * Does this dialect/database support non-query statements (e.g. INSERT, UPDATE, DELETE) in CTE (Common Table Expressions)?
+	 *
+	 * @return {@code true} if CTE are supported
+	 */
+	public boolean supportsNonQueryInCTE() {
+		return false;
+	}
+
+	/**
+	 * Does this dialect/database support VALUES list (e.g. VALUES (1), (2), (3) )
+	 *
+	 * @return {@code true} if CTE are supported
```

Comment:
Sorry for not being specific, I meant that in part `@return {@code true} if CTE are supported` should it not be `VALUES` instead of `CTE`?